docs/Clearwater_Configuration_Options_Reference.md

```diff
@@ -110,6 +110,8 @@ This section describes optional configuration options, particularly for ensuring
 
 * `homestead_provisioning_port` - the HTTP port the Homestead provisioning interface listens on. Defaults to 8889. Not needed when using an external HSS.
 * `sas_server` - the IP address or hostname of your Metaswitch Service Assurance Server for call logging and troubleshooting. Optional.
+* `snmp_ip` - this determines where SNMP alarms get sent, and is a comma-separated list of IP addresses (e.g. `snmp_ip=0.0.0.0,0.0.0.1`).
```

Comment:
snmp_ip is already defined in this file. Can you move the snmp_notification_types definition by the existing snmp_ip one?